cms/djangoapps/contentstore/tasks.py

```diff
@@ -66,3 +67,12 @@ def deserialize_fields(json_fields):
     for field_name, value in fields.iteritems():
         fields[field_name] = getattr(CourseFields, field_name).from_json(value)
     return fields
+
+
+@task()
+def publish_bulk_notifications_task(user_ids_list, notification_msg):
+    """
+    This function will call the edx_notifications api method "bulk_publish_notification_to_users"
+    and run as a new Celery task.
+    """
+    bulk_publish_notification_to_users(user_ids_list, notification_msg)
```

Comment:
Actually, if we are going to use Celery, then I think we have to have the Django ORM query be *inside* the task itself, otherwise, Celery is going to have to serialize potential 100,000's of user_ids. So I think the call signature needs to be here:

def publish_course_notifications_task(course_id, notification_msg):

Also, we'll probably need to re-use this "broadcast" functionality in the LMS (Instructor Dashboard) at some point. I don't know a lot of our use of Celery, but can we move this task to be inside "common" instead, so we can use it in both the LMS and Studio.